support-e2e/tests/tieredCheckout.test.ts

```diff
@@ -122,22 +122,22 @@ const testDetailsPromo = [
 		frequency: 'Monthly',
 		promoCode: 'E2E_TEST_SPLUS_MONTHLY',
 		expectedCardHeading: 'All-access digital',
-		expectedPromoText: '£8/month for 6 months, then £10/month',
-		expectedCheckoutTotalText: 'Was £10, now £8/month',
-		expectedCheckoutButtonText: 'Pay £8 per month',
+		expectedPromoText:
+			/£(\d|\.)+\/month for (\d|\.) months, then £(\d|\.)+\/month/,
+		expectedCheckoutTotalText: 'Was £12, now £9.60/month',
```

Comment:
```suggestion
			expectedCheckoutTotalText: /Was £(\d|\.)+, now £(\d|\.)+\/month/,
```

Reply:
Do we want to hard code the S+ monthly amounts test?

Reply:
Nope, to accommodate price raises across `CODE` and `PROD`